index.jsx

```diff
@@ -106,6 +107,14 @@ export default {
         }),
         onToggleLED: () => dispatch(RssiActions.toggleLED()),
     }),
+    mapSidePanelState: (state, props) => ({
+        disabled: !!(state.app.port === null),
```

Comment:
the === comparison will always evaluate to a boolean, so why use the double not operator here?